code/modules/jobs/job_types/roguetown/garrison/dungeoneer.dm

```diff
@@ -58,3 +58,4 @@
 	H.dna.species.soundpack_m = new /datum/voicepack/male/warrior()
 	H.verbs |= /mob/living/carbon/human/proc/torture_victim
 	ADD_TRAIT(H, TRAIT_MEDIUMARMOR, TRAIT_GENERIC)
+	ADD_TRAIT(H, TRAIT_DEATHBYSNUSNU, TRAIT_GENERIC)
```

Comment:
add a newline at the end of the file

Reply:
well. there is a new line when i try to edit it, i dunno.